### DISCUSSION TITLE
What to reply for arcade/colonnade "entrances"?

### DISCUSSION THREAD

**Discussion by ivanbranco** [streetcomplete/StreetComplete#4599] [Q&A] [answered]
Hi,

StreetComplete asked me what kind of entrance is this one: https://www.openstreetmap.org/node/967793522/history
I attach image of this "entrance": ![CANCELLARE](https://user-images.githubusercontent.com/94002623/199500957-e94a8fa7-d322-4720-af35-ce129ae0ca93.jpg)

But I wasn't sure about it. Should it be tagged as an "entrance" to begin with (since you dont get into the actual building)? If yes, what kind? A simple entrance=yes?

**Comment by matkoniecz** [comment, marked as answer]
This quest should skip such arcades. So if it is being asked it is always mistake in OSM data or bug in StreetComplete.

In this case steps were without `covered` tag which I fixed in https://www.openstreetmap.org/changeset/128394076

So you should create a note with photo or edit it with Vespucci or other editor in case of quest being asked in such places.

**Reply by valerio-bozzolan** [reply]
> This quest should skip such arcades.

Agree. Is there an issue about it?

**Reply by matkoniecz** [reply]
"should", as in "already does it"

So you should create a note with photo or edit it with Vespucci or other editor in case of quest being asked in such places.

If you are unsure how, please link affected location.
